src/molecule/app.py

```diff
@@ -0,0 +1,13 @@
+"""Molecule Application Module."""
+from ansible_compat.runtime import Runtime
+
+
+class App:
+    """App class that keep runtime status."""
+
+    def __init__(self) -> None:
+        """Create a new app instance."""
+        self.runtime = Runtime(isolated=True)
+
+
+app = App()
```

Comment:
Is this an attempt to make a mutable singleton? Why not initialize it during the startup sequence and then pass around DI-style?

Reply:
Molecule multi-layered structure makes it extremely inconvenient to pass such an object. I considered that approach but it would be a total mess. We can aim to achieve that at some point but it would need several major version changes and existence of plugins is a major limiting factor.

Reply:
Affecting the import time is also messy. There's nothing wrong with multilayered bypasses, though.

Reply:
I will not do that as part of this PR. If you want you can lead this challenge but I warn you that you will unlikely to succeed as you will have to change API of plugins and 15+ plugins. I am also worried that this may add delays which may interfere with `AAP` inclusion goals.